tests/bootstrap.php

```diff
@@ -0,0 +1,2 @@
+<?php
+@include_once dirname(__FILE__) . '/../vendor/autoload.php';
```

Comment:
Need a newline at the end of this file at the least.

What does this and the change to `phpunit.xml` accomplish?  Is autoload generated by composer?  I'd rather not be dependent on composer just to run the tests if we can manage it.